src/routes/Dashboard/components/PopUpCard/PopUpCard.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Paper from 'material-ui/Paper';
+import { Toolbar, ToolbarTitle } from 'material-ui/Toolbar';
+import classes from './PopUpCard.scss';
+
+
+const styles = {
+  paper: {
+    height: 300,
+    width: 220,
+    margin: 20,
+    overflow: 'scroll',
+  },
+};
+
+
+const PopUpCard = (props) => {
+  let currentLocation;
+  let estimatedTimeOfArrival;
+  let updatedAt;
+  if(props.shipment){
+    const shipment = props.shipment;
+    if (shipment.currentLocation) {
+      currentLocation = (
+        <div><h6>CURRENT LOCATION</h6> {shipment.currentLocation.city},
+          {shipment.currentLocation.state}</div>
+      );
+    }
+    if (shipment.estimatedTimeOfArrival) {
+      estimatedTimeOfArrival = (
+        <div><h6>ESTIMATED TOA</h6> {shipment.estimatedTimeOfArrival}</div>
+      );
+    }
+    if (shipment.updatedAt) {
+      updatedAt = (
+        <div><h6>LAST UPDATED</h6> {shipment.updatedAt}</div>
+      );
+    }
+  }
+  const title = props.shipment
+    ? `Shipment ${props.shipment.id}`
+    : '...';
+  return (
+    <Paper style={styles.paper} zDepth={2}>
+      <Toolbar>
+        <ToolbarTitle text={title} />
+      </Toolbar>
+      <div className={classes.mainSection}>
+          <div className={classes.col1}>
+            <h6>ORDER</h6>
+              { props.shipment ? props.shipment.id : '...'}{"\n"}
+          </div>
+          <div className={classes.col2}>
+            <h6>STATUS</h6>
+            { props.shipment ? props.shipment.status : 'loading...'}{"\n"}
+          </div>
+
+          { currentLocation }
+          { estimatedTimeOfArrival }
+          { updatedAt }
+
+
+      </div>
+    </Paper>
+  );
+};
+
+PopUpCard.propTypes = {
```

Comment:
We can refactor this later if you want, but it's probably more helpful to assume nothing about the API giving the data in the component, and ask for props exactly as it would be helpful to render them.

For example, you may want to ask for an object like:
```
shipment: {
  location: 'City, State',
  lastUpdated: 'time',
  eta: 'time',
  orderId: 123,
  status: 'pending',  
}
```

and then, also we want to do a full prop validation for the object coming in. Here is how I did it for the role switcher:
```
RoleItem.propTypes = {
  user: React.PropTypes.shape({
    id: React.PropTypes.number.isRequired,
    role: React.PropTypes.string.isRequired,
    location: React.PropTypes.string,
    loggedIn: React.PropTypes.bool,
  }),
  roleAction: React.PropTypes.func.isRequired,
};
```